tandem/simulate/rmg_simulator.py

```diff
@@ -0,0 +1,311 @@
+"""
+RMG Simulator Adapter module
+Used to run mechanism analysis with RMG
+"""
+
+import os
+import pandas as pd
+import shutil
+from typing import Optional, Union
+
+from rmgpy.exceptions import InputError
+from rmgpy.solver.simple import SimpleReactor
+from rmgpy.solver.liquid import LiquidReactor
+from rmgpy.species import Species
+from rmgpy.tools.loader import load_rmg_py_job
+from rmgpy.tools.simulate import simulate
+
+from tandem.logger import log
+from .adapter import SimulateAdapter
+from .factory import register_simulate_adapter
+
+
+class RMGSimulator(SimulateAdapter):
+    """
+    RMGSimulator is an adapter for the abstract class SimulateAdapter. All output is stored in an "rmg_analysis"
+    directory inside the RMG-ARC iteration directory.
+
+    Args:
+        run_directory (str): The path to the RMG-ARC iteration directory.
+        atol (Optional[float]): The absolute tolerance used when running a simulation.
+        rtol (Optional[float]): The relative tolerance used when running a simulation.
+        observable_list (Optional[list]): Species used for SA. Entries are dictionaries of 'label' and structure
+                                          (either 'smiles' or 'adj'). SA is run only if this argument is specified.
+        sa_threshold (Optional[float]): The sensitivity threshold to use.
+        sa_atol (Optional[float]): The absolute tolerance used when running SA.
+        sa_rtol (Optional[float]): The relative tolerance used when running SA.
+        verbose (Optional[bool]): Whether or not to log to file.
+
+    ======================= ======      ====================================================
```

Comment:
we don't use this style for attributes in T3 (we do in RMG), you can have an `Attributes:` section in a similar format to `Args:`

Reply:
Thanks for reviewing! I just pushed the updated format for documenting the attributes. I think the pyrms simulator adapter is ready as well, just needs SA to be working in rms. I'm happy to help with the T3 input file in the meantime :)